README.md

```diff
@@ -148,7 +148,7 @@ For security, you need to enter something into the `SECRET_KEY`.
 SECRET_KEY = 'somethingnotempty'
 ```
 
-By default, you need a `Memcache` server up and running. The connection
+*By default, you need a `Memcache` server up and running.* The connection
```

Comment:
Italic text is for emphasis on a specific word like "The cat was in the *house* not in the barn". 
Use bold for things that are supposed to stand out.